hooks/post_gen_project.py

```diff
@@ -25,5 +25,6 @@
 
 4. Register your app NOW by making a pull request to the AiiDAlab App Registry
 on https://github.com/aiidalab/aiidalab-registry
-See this link also for a full list of keys for your metadata.json file.
+
+See this link also for a full list of recognized metadata keys for the setup.cfg file.
```

Comment:
shouldn't you list to the aiidalab documentation now instead?

Reply:
good catch, thanks!